src/main/webapp/features/screens/game/dashboard/gameMap/ct-game-map.js

```diff
@@ -10,6 +14,8 @@ angular.module('catan')
                 },
                 link: function(scope, element) {
 
+                    var hexesHighlightTimeout = null;
```

Comment:
Hexes are animated when dice value is changed. If some animation starts when previous one is not finished yet, we should stop the previous one. Animation is provided with javascript timeout. "hexesHighlightTimeout" - is a "promise" which is using for stoping of timeout before a new timeout (animation) needs to be started.

Why is it here? Because it is a private attribute of each instance of the directive.